satellite/satellitedb/overlaycache.go

```diff
@@ -1022,6 +1022,23 @@ func (cache *overlaycache) GetExitingNodesLoopIncomplete(ctx context.Context) (e
 	}
 	return exitingNodes, nil
 }
+func (cache *overlaycache) GetExitStatus(ctx context.Context, nodeID storj.NodeID) (_ *overlay.ExitStatus, err error) {
```

Comment:
Would there be any issue with just using `Get(ctx, id)` rather than adding a new function for `GetExitStatus`? `Get` uses `convertDBNode` so it should be possible to use it to get the exit status for a specific node. I guess the main downside is that it would be retrieving a bunch of other node data that we don't care about. I don't know what kind of performance impact that has, but we should weigh it against this option.

Reply:
I was just trying to make the lookup as lean as possible. Performance impact is probably minimal in this scenario though.  I can change, but I also don't see the harm in only retrieving what is needed.  I'm torn :)

Reply:
I like that it's clean. I think it's fine to keep.